Library/Formula/tidy-html5.rb

```diff
@@ -0,0 +1,20 @@
+class TidyHtml5 < Formula
+  homepage "http://www.html-tidy.org/"
+  url "https://github.com/htacg/tidy-html5/archive/4.9.26.tar.gz"
+  sha256 "28674745db53b6ef1aa4b8466e6e231915dcd596672ec40515d0ab53ee0c33f6"
+
+  depends_on "cmake" => :build
+
+  def install
+    cd "build/cmake"
```

Comment:
actually it is, according to tidy's docs. Can confirm: removing that line will cause a build failure.

```diff
  def install
    system "cmake", *std_cmake_args
    system "make"
    system "make", "install"
  end
```

```
==> Downloading https://github.com/htacg/tidy-html5/archive/4.9.26.tar.gz
Already downloaded: /Library/Caches/Homebrew/tidy-html5-4.9.26.tar.gz
==> cmake -DCMAKE_C_FLAGS_RELEASE= -DCMAKE_CXX_FLAGS_RELEASE= -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/tidy-html5/4.9.26 -DCMAKE_BUILD_T
==> make
warning: failed to load external entity "../documentation/tidy1.xsl"
cannot parse ../documentation/tidy1.xsl
make[2]: *** [man] Error 4
make[1]: *** [CMakeFiles/man.dir/all] Error 2
make: *** [all] Error 2
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1

READ THIS: https://git.io/brew-troubleshooting
```